### ISSUE TITLE
License?

### ISSUE THREAD

**Issue by alogic0** [HermannBjorgvin/Gravity-Experiments#19]
Thank you for interesting and educational project. What a License for using and changing your code?  You didn't choose one.

**Comment by HermannBjorgvin** [comment]
It's not licensed right now. I have to get more familiar with different types of licenses before I can pick one. You can use it for whatever you want though

**Comment by alogic0** [comment]
Many thanks for this personal permission! But I'm concerning about other users too. By default [they are restricted](https://help.github.com/articles/licensing-a-repository/) to only viewing and forking code. I even not sure that forking means permission to change. So it would be better to choose any more permitable license ([MIT](https://choosealicense.com/licenses/mit/), etc.) for now and change it by the time by any other. As an author you have a full right to do that.

**Comment by HermannBjorgvin** [comment]
Yes Alogic0, I'm looking into the specific differences between MIT and GPL2 and other licenses. I prefer to spend my time coding rather than looking at licenses but yes it is important to have a license that represents your will.
